PhpcsChanged/GitWorkflow.php

```diff
@@ -92,7 +95,7 @@ function isNewGitFileLocal(string $gitFile, string $git, callable $executeComman
 	$gitStatusOutput = $executeCommand($gitStatusCommand);
 	$debug('git status output:', $gitStatusOutput);
 	if (! $gitStatusOutput || false === strpos($gitStatusOutput, $gitFile)) {
-		throw new ShellException("Cannot get git status for file '{$gitFile}'");
+		return false;
```

Comment:
I'm not sure how this worked before, since `git status file.php` will be an empty string for an already existing file.